package.json

```diff
@@ -5,6 +5,9 @@
   "dependencies": {
     "@ant-design/icons": "^4.2.2",
     "@emotion/core": "^10.0.35",
+    "@emotion/react": "^11.1.5",
+    "@material-ui/core": "^4.11.3",
```

Comment:
Is this used? If not, can we `yarn remove` it?

Reply:
I know we need emotion/react

tried to remove material-ui/core but could not get the icons to work nicely without it.